assets/translations/zh-hans.yaml

```diff
@@ -0,0 +1,1801 @@
+##############################
+# Special stuff
+##############################
+generic:
+  add: 添加
+  back: 返回
+  cancel: 取消
+  clickToShow: "[显示值]" 
+  close: 关闭
+  comingSoon: 即将推出
+  create: 创建
+  created: 创建时间 
+  customize: 定制 
+  default: 默认
+  disabled: 禁用
+  enabled: 启用
+  enforced: Enforced # no use
+  ignored: 忽略
+  labelsAndAnnotations: 标签和注释
+  na: n/a
+  name: 名称
+  none: 无
+  number: '{prefix}{value, number}{suffix}'
```

Comment:
Anything that doesn't need translation (like this one), or is identical in both languages should be removed from here.  Or commented out, if you intend to translate it later and it's easier to organize what's still left to do that way.

The English value is automatically used if a translation is missing for the selected language, and it's easier to search for and maintain one copy of that than many.